wbc_pr2_ctrl:
  tao_root_name: torso_lift_link
  active_links:
   - l_shoulder_pan_link
   - l_shoulder_lift_link
   - l_upper_arm_roll_link
   - l_elbow_flex_link
   - l_forearm_roll_link
   - l_wrist_flex_link
   - l_wrist_roll_link
  gravity_compensated_links:
   - l_shoulder_pan_link
   - l_shoulder_lift_link
   - l_upper_arm_roll_link
   - l_upper_arm_link
   - l_elbow_flex_link
   - l_forearm_roll_link
   - l_forearm_link
   - l_wrist_flex_link
   - l_wrist_roll_link
  gains:
   l_shoulder_pan_joint:   { kp: 800, kd: 10 }
   l_shoulder_lift_joint:  { kp: 800, kd: 10 }
   l_upper_arm_roll_joint: { kp: 200, kd:  3 }
   l_elbow_flex_joint:     { kp: 100, kd:  3 }
   l_forearm_roll_joint:   { kp: 500, kd:  6 }
   l_wrist_flex_joint:     { kp: 400, kd:  4 }
   l_wrist_roll_joint:     { kp: 400, kd:  4 }
opspace_task_posture_controller:
  dt_seconds:     0.002
  end_effector:   l_wrist_roll_link
  control_point:  [   0.25,  0.0,   0.0 ]
  task_goal:      [   0.5,   0.0,   1.0 ]
  task_kp:        [ 100.0, 100.0, 100.0 ]
  task_kd:        [  20.0,  20.0,  20.0 ]
  task_maxvel:    [   0.6,   0.6,   0.6 ]
  task_maxacc:    [   1.2,   1.2,   1.2 ]
  posture_goal:   [   0.0,   0.0,   0.0,   0.0,   0.0,   0.0,   0.0 ]
  posture_kp:     [ 800.0, 800.0, 200.0, 100.0, 500.0, 400.0, 400.0 ]
  posture_kd:     [  10.0,  10.0,   3.0,   3.0,   6.0,   4.0,   4.0 ]
  posture_maxvel: [   1.2,   1.2,   1.2,   1.2,   1.2,   1.2,   1.2 ]
  posture_maxacc: [   2.4,   2.4,   2.4,   2.4,   2.4,   2.4,   2.4 ]
